plugins/collections/get.js

```diff
@@ -6,6 +6,42 @@ const schema = require('screwdriver-data-schema');
 const getSchema = schema.models.collection.get;
 const idSchema = joi.reach(schema.models.collection.base, 'id');
 
+/**
+ * Helper function to get last builds of pipelines
+ *
+ * @param   {Array}    pipelines     - array of pipeline models to get last builds for
+ * @param   {Factory}  eventFactory  - factory for getting last event model
+ * @returns {Array}
+ */
+function getPipelinesHealth(pipelines, eventFactory) {
+    return pipelines
+        .filter(pipeline => !!pipeline)
+        .map((pipeline) => {
+            const result = Object.assign({}, pipeline.toJson());
+
+            if (!pipeline.lastEventId) {
+                return result;
+            }
+
+            return eventFactory.get(pipeline.lastEventId)
+                .then((event) => {
+                    if (!event) {
+                        return result;
+                    }
+
+                    return event.getBuilds()
+                        .then((builds) => {
+                            if (builds.length) {
+                                result.lastBuilds = builds.map(b => b.toJson()).reverse();
```

Comment:
The order of the lastBuilds is being reversed here because `getBuilds()` returns the builds in the order of the most recent build being first. Reversing the order will allow us to map the builds to the workflow accurately on the UI  for displaying on the dashboard.